### PR TITLE
release-helper: Print responses from GH api calls

### LINKED ISSUE #25: release-helper: Print GitHub PR/release hyperlink after creating it
This would make it easier for the foreperson to double-check if everything was created fine.

### PR DESCRIPTION
**Pull request by ochosi**
Fixes #25

I tested this with a dummy repository in the python interactive shell (octocat/Hello-World).

### CODE REVIEW

File: release.py [hunk 1]
```diff
@@ -335,13 +335,21 @@ def create_pullrequest(args, api):
     head = f'{args.user}:release-{args.version}'
     body= 'Tasks:\n- [ ] Bump version\n- [ ] Update news'
 
-    api.pulls.create(title, head, args.base, body, True, False, None)
+    try:
+        res = api.pulls.create(title, head, args.base, body, True, False, None)
+        msg_ok(f"Pull request successfully created: {res.html_url}")
+    except: # pylint: disable=bare-except
```
Comment (diaasami) [inline release.py:341]: May be receive an exception object `e` and print it after the error message so that there is a bit of context why things failed.

**Review by diaasami — COMMENTED** [review]
Just that 1 comment, otherwise looks good.

File: release.py (hunk 1 above)
Comment (ochosi) [reply release.py]: Ok that makes sense, with the small problem that the error message also triggers a `sys.exit(1)`, so I added the print statement above it.

**Review by msehnout — APPROVED** [review]
LGTM 👍

**Pull request merged**